src/mcp_agent/workflows/llm/multipart_converter_azure.py

```
@@ -0,0 +1,360 @@
+            multipart_msg: The PromptMessageMultipart message to convert
+
+        Returns:
+            An Azure UserMessage or AssistantMessage object
+        """
+        role = multipart_msg.role
+
+        if not multipart_msg.content:
+            if role == "assistant":
+                return AssistantMessage(content="")
+            else:
+                return UserMessage(content="")
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)